x-pack/test/api_integration/apis/security_solution/timeline_migrations.ts

```diff
@@ -44,14 +46,14 @@ export default function ({ getService }: FtrProviderContext) {
         await esArchiver.load(
           'x-pack/test/functional/es_archives/security_solution/timelines/7.15.0_space'
         );
-      });
-
-      after(async () => {
-        await esArchiver.unload(
-          'x-pack/test/functional/es_archives/security_solution/timelines/7.15.0_space'
+        await kibanaServer.importExport.load(
+          'x-pack/test/functional/fixtures/kbn_archiver/security_solution/timelines/7.15.0_space',
+          { space: 'awesome-space' }
         );
       });
 
+      after(async () => await spacesService.delete('awesome-space'));
```

Comment:
Should we add the `esArchiver.unload()` here?

Reply:
Hrmm, well if they archives are loading into space then no....
Lemme triple check as you may be onto something.

Reply:
Ok, good thing you had me check!

I noticed two objects that should not have been within the archive. (awesome!)
Also, yes we should still be good only dropping the space. 
This is because all the archives are loaded into that space && deleting a space drops all of the objects.

> TLDR

_Since it's on disk as  a gzipped archive, I ran the following and then visually inspected it_
`node scripts/es_archiver.js edit x-pack/test/functional/es_archives/security_solution/timelines/7.15.0_space --config x-pack/test/api_integration/config.ts`


```
jq '[.value.id]| .[]' x-pack/test/functional/es_archives/security_solution/timelines/7.15.0_space/data.json
"awesome-space:siem-ui-timeline:1211b430-25f8-11ec-a981-b77847c6ef30"
"awesome-space:siem-ui-timeline:120f9150-25f8-11ec-a981-b77847c6ef30"
"awesome-space:siem-ui-timeline:12205a30-25f8-11ec-a981-b77847c6ef30"
"awesome-space:siem-ui-timeline:12208140-25f8-11ec-a981-b77847c6ef30"
"awesome-space:siem-ui-timeline:1e2e9850-25f8-11ec-a981-b77847c6ef30"
"awesome-space:siem-ui-timeline-note:2d230670-25f8-11ec-a981-b77847c6ef30"
"awesome-space:siem-ui-timeline-pinned-event:2694b600-25f8-11ec-a981-b77847c6ef30"
"awesome-space:siem-ui-timeline-note:323df0c0-25f8-11ec-a981-b77847c6ef30"
```

Unless I missed my guess (totally possible lol), I think we are good as they are "namespaced" to `awesome-space`.